### PR TITLE
mssql-source:upgrade debezium version to 1.9.6

### LINKED ISSUE #15248: Upgrade MSSQL debezium version to 1.9.6
Required for beta (heading is obvious)

### PR DESCRIPTION
**Pull request by subodh1810**
Issue : https://github.com/airbytehq/airbyte/issues/15248

Tested this locally.
1. Created a brand new connector via image built with these changes. Both first sync and subsequent syncs worked as expected.
2. Created a connector on version 0.4.24 and ran a sync and then changed the connector to use the dev image which included changes from this PR and the sync continued to work as expected proving this is backward compatible.

### CODE REVIEW

**Comment by akashkulk** [comment]
Thanks Subodh - could you quickly summarize the main changes in the PR description in this PR, and the ones needed to happen to upgrade the ms-sql connector to dbz 1.9.6?

File: airbyte-integrations/bases/debezium-v1-9-6/build.gradle [hunk 1]
```diff
@@ -11,7 +11,7 @@ dependencies {
 
     implementation 'io.debezium:debezium-api:1.9.6.Final'
     implementation 'io.debezium:debezium-embedded:1.9.6.Final'
-//    implementation 'io.debezium:debezium-connector-sqlserver:1.9.2.Final'
+    implementation 'io.debezium:debezium-connector-sqlserver:1.9.6.Final'
```
Comment (rodireich) [inline airbyte-integrations/bases/debezium-v1-9-6/build.gradle:14]: I just wanted to make sure that what was forgotten in Postgres (`TINYINT(1)`?) is taken care of here.

**Review by rodireich — APPROVED** [review]

**Review by akashkulk — APPROVED** [review]

**Pull request merged**